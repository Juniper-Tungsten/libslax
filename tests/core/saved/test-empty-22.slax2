/* Machine Crafted with Care (tm) by slaxWriter */
version 1.1;

match / {
    <top> {
        var $x = /test;
        
        if (not($x) && not(/foo/bar) && not(goo[zoo && not(moo)])) {
            <not>;
        }
        var $y = 10;
        var $z = slax:evaluate("$y + 1");
        <answer> {
            copy-of $z;
        }
        <answer> slax:evaluate("!$x");
    }
}
