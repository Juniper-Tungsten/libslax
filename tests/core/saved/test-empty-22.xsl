<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:foo="http://example.com/foo" xmlns:slax="http://code.google.com/p/libslax/slax" version="1.0" extension-element-prefixes="foo slax">
  <xsl:template match="/">
    <top>
      <xsl:variable name="x" select="/test"/>
      <xsl:if test="not($x) and not(/foo/bar) and not(goo[zoo and not(moo)])">
        <not xmlns:goo="http://example.com/goo" xsl:extension-element-prefixes="goo">
          <goo:this>oh yeah</goo:this>
        </not>
      </xsl:if>
      <mumble xmlns:zoo="http://example.com/zoo" xmlns:yoo="http://example.com/yoo" xsl:extension-element-prefixes="zoo yoo">
        <zoo:zoo/>
        <yoo:yoo/>
      </mumble>
      <xsl:variable name="y" select="10"/>
      <xsl:variable xmlns:slax="http://code.google.com/p/libslax/slax" name="z" select="slax:evaluate(&quot;$y + 1&quot;)"/>
      <answer>
        <xsl:copy-of select="$z"/>
      </answer>
      <answer>
        <xsl:value-of xmlns:slax="http://code.google.com/p/libslax/slax" select="slax:evaluate(&quot;!$x&quot;)"/>
      </answer>
      <xsl:value-of xmlns:slax="http://code.google.com/p/libslax/slax" select="slax:output(&quot;this &quot;, &quot;is &quot;, &quot;a &quot;, &quot;test&quot;)"/>
    </top>
  </xsl:template>
</xsl:stylesheet>
