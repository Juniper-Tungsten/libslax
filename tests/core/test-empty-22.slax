version 1.1;

match / {
    <top> {
	var $x = /test;
	if (!$x && !/foo/bar && !goo[zoo && !moo]) {
	    <not>;
	}
    }
}
