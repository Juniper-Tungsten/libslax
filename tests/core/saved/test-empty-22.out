<?xml version="1.0"?>
<top>
  <not/>
  <answer>11</answer>
  <answer>true</answer>
</top>
